schedule/urls.py

```diff
@@ -3,6 +3,9 @@
 
 
 urlpatterns = [
-#
+    url(r'^$', views.log_in, name='log_in'),
+
+    url(r'^base/(?P<uid>\d+)/$', views.base, name='base'),
+    url(r'^room_page/(?P<rid>\d+)/$', views.room_page, name='room_page'),
```

Comment:
I like trailing slashes in the end of URL. And it is also a good practice to use names!